tests/dualtor/test_tor_ecn.py

```diff
@@ -334,11 +337,8 @@ def test_ecn_during_decap_on_active(
 
 def test_ecn_during_encap_on_standby(
```

Comment:
Do we need to ensure that the ToR selected for the test is standby? i.e. using `apply_standby_state_to_orchagent` similar to above tests

Reply:
Done